lib/Dialect/FIRRTL/Transforms/GrandCentralTaps.cpp

```diff
@@ -529,7 +542,7 @@ LogicalResult static applyNoBlackBoxStyleDataTaps(const AnnoPathValue &target,
             [&](auto v) { return sinkBuilder.create<AsAsyncResetPrimOp>(v); });
     }
 
-    state.wiringProblems.push_back({sendVal, sink, nameHint});
+    state.wiringProblems.push_back({sendVal, sink, ""});
```

Comment:
Ah, does this mean "generate a good name for me"? Awesome!

Reply:
Yep! If this `nameHint` is _not_ provided, then it's telling the `WiringProblem` to figure out a good name. The nameHint escape hatch exists if a good name is not discernible.